src/options/flags.rs

```diff
@@ -82,6 +82,7 @@ pub static OCTAL:             Arg = Arg { short: Some(b'o'), long: "octal-permis
 pub static SECURITY_CONTEXT:  Arg = Arg { short: Some(b'Z'), long: "context",              takes_value: TakesValue::Forbidden };
 pub static STDIN:             Arg = Arg { short: None,       long: "stdin",                takes_value: TakesValue::Forbidden };
 pub static FILE_FLAGS:        Arg = Arg { short: Some(b'O'), long: "flags",                takes_value: TakesValue::Forbidden };
+pub static INSPECT_ARCHIVES:  Arg = Arg { short: Some(b'q'), long: "quasi",                takes_value: TakesValue::Forbidden };
```

Comment:
MAy I ask the reason of `--quasi` ? I think its okay as a name but I don't get the link with archives

Reply:
It's derived from AIX's `li` program, to treat archives as "quasi-directories".

Reply:
Great

Reply:
I think I'd prefer this to be `--inspect-archives`

Reply:
I don't have a preference tbh so its up to you all

Reply:
I also don't have a strong opinion, although I think having a short flag for it would be great - but in the end, this can also be solved via a shell alias.
Maybe :tada: for `--quasi` and :rocket: for `--inspect-archives` (or other suggestions)?

Reply:
```c
/*                                                                          
 * li - list files, directories, archives                                        
 *                                                                          
   Rewritten by D. Landauer to combine the functions of seven different     
   `ls' programs we had here (May '79) ... also made it use stdio.  Flags:  
        -I<abcfghilmnoprsu>    include a field                              
        -E<abcfghilmnoprsu>    exclude a field                  **rrr**     
            accessed-time      headers       modified-time  protections     
            character count    i-number      name           size (in blocks)
            group id           link count    owner id       updated-time    
        -O<abcdfpsx> list only these types of files                         
            archives           directories           symbolic links         
            block devices      files (normal)        executable files       
            character devices  pipes (fifos)                                
                                                                            
        -S[r]<acmnsux> sort order [r= reversed]                             
            accessed-time (latest first)            name (default)          
            modified-time (latest first)            size (biggest first)    
            updated-time  (latest first)            x = no sort at all      
        -R[<n>anqp] recursive, to level n                                   
            a absolute pathnames                                            
            n don't list qdirs' contents (default)                          
            p pathnames (relative)                                          
            q do list qdirs' contents                                       
        -<n>    at most n columns                                           
        -a      all files                                                   
                     VMS:   all versions of files                           
                     other: even ones beginning w/ `.'                      
        -d      directories (not their contents)                            
        -f      force interpretation as a directory                         
        -l      local long form  (-Icglmnop)                                
        -k      remote long form (-Ibcfmnpr)            * removed *         
        -n      real file names:  do not make them printable (==> -1)       
        -s      sortable verbose (Berklix style)                            
        -v      visual/verbose                                              
        -x      extended (-Iabcfgilmoprsu)                                  
        -L      follow symbolic links (default is to not follow)            
                                                                            
   Routines are in alpha order except main & init (the first two) and       
   getpwn, getgrpn and search (the last three).   Throughout, I refer to    
   archives as "quasi-directories", often abbreviated as qdirs.             
   (Quasi-directories are a superset of directories.)                       
                                                                            
   Because of the removal of the Distribution Services, the following       
   options will be affected:                                                
        1) -k option is removed.                                            
        2) -Ibfr options are ignored, but it will display '-'s instead.     
        3) -Ebfr options are ignored.                                       
        4) -x option is affected. It will display '-' for -bfr fields.      
 */                                                                         
```

The IBM `li` program that treats archives as "quasi-directories" goes back to the late 70's at least, so maybe it's in some people muscle memory.  While I do like `-q`/`--quasi` myself, it's not something I'm too worried about. :)